assets/src/edit-story/app/story/actions/useSavePost.js

```diff
@@ -1,13 +1,40 @@
 /**
  * WordPress dependencies
  */
-import { useCallback } from '@wordpress/element';
+import { useCallback, renderToString } from '@wordpress/element';
 import { addQueryArgs } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
 import { useAPI } from '../../api';
+import { getDefinitionForType } from '../../../elements';
+
+/**
+ * Creates AMP HTML markup for saving to DB for rendering in the FE.
+ *
+ * @param {Object} pages Object of pages.
+ * @return {Element} Markup of pages.
+ */
+const getStoryMarkupFromPages = ( pages ) => {
+	// @todo look into: We should rather use renderToStaticMarkup here, however, it doesn't seem to be exposed via @wordpress/element.
```

Comment:
`wp.element.renderToString` seems to be a custom implementation, and not just an alias for `ReactDOMServer.renderToString`:

https://github.com/WordPress/gutenberg/blob/67cb1cc7d85e5134c580fba0ed4b362a5d95e07f/packages/element/src/serialize.js#L348-L419

Reply:
Oh! Good catch, thanks for checking that. Perhaps that's what we need then, will check.

Reply:
Looks like that works!